ducktape/tests/runner.py

```diff
@@ -210,6 +210,9 @@ def run_all_tests(self):
                         self._log(logging.ERROR, err_str)
 
                         # All processes are on the same machine, so treat communication failure as a fatal error
+                        for proc in self._client_procs.values():
+                            proc.terminate()
```

Comment:
Also compare this to https://github.com/confluentinc/ducktape/blob/master/ducktape/tests/runner.py#L124 which uses `os.kill` vs `terminate()` - what's the difference and pros/cons?

Reply:
would probably be good to have a unified `cleanup_child_processes` method or smth

Reply:
if you read the docs for terminate:
```
Terminate the process. On Unix this is done using the SIGTERM signal; on Windows TerminateProcess()
``` 
which seems to be more platform agnostic

Reply:
What about modifying that other line too then? Can be a separate PR though.

Reply:
i can go ahead and add it here